unlock-protocol.com/blog/unlock-with-credit-cards.md

```diff
@@ -0,0 +1,30 @@
+authorName: Julien Genestoux
+publishDate: April 16, 2020
+description: Unlock memberships with credit cards, whether you use a crypto wallet, or not!
+image: /static/images/blog/unlock-user-accounts/user-accounts.jpg
+---
+
+The blockchain movement started with crypto-currencies. Bitcoin became the first permission-less and decentralized money. Money balances are "state" stored in a ledger. Ethereum went further by becoming the first blockchain which can store any kind of "state". At Unlock, we think it's a great [platform to store memberships](https://unlock-protocol.com/blog/protocol-for-membership/).
+
+About 6 months ago, we introduced our [Unlock accounts](https://unlock-protocol.com/blog/unlock-user-accounts/), which are easy ways for people who do no have a crypto wallet, or not do own crypto-currencies to easily and **instantly** purchase their memberships. We hide the "crypto" complexity under the hood (seed phrases, gas fees... etc) so that internet users can easily Unlock content, without having to learn new paradigms... until [they actually want to](https://unlock-protocol.com/blog/ejecting-account/)!
+
+Today, I am very proud to introduce the next iteration of our work on supporting credit cards. Starting now, **crypto users can also purchase memberships using their credit cards** on selected locks.
+
+We believe that wallets are great to store money and access keys, but that they are also a very good identity mechanism. In other words, you should be able to [login into any website using your wallet](https://www.nickgrossman.is/2020/digital-bearer-assets/), rather than using an email address and password. Whether the wallet actually holds money could almost be irrelevant. As such, we think memberships should be attached to the wallet, whether they have actually been purchased _from_ the wallet, or using another payment mechanism, such as credit cards!
+
+1. When a user finds a lock with credit cards enabled, they can select _Credit Card_ as their payment mechanism.
+2. The price will be slightly higher to cover credit card fees.
+3. The user is then asked to enter their credit card number and sign it with their crypto wallet
+4. Once submitted, our backend will then charge the card, and issue the corresponding NFT!
+
+This is by far the easiest way to purchase your first crypto asset. Want to try by yourself? Check these locked sites:
+
+- [Livepeer dashboards on Scout](https://medium.com/@scout_cool/announcing-livepeer-orchestrator-dashboard-with-an-unlock-fca60750843b), like [this one](https://scout.cool/livepeer/mainnet/dashboards/5e28c582bfbf6c00177109e0?&nodeaddress=0xE9E284277648fcdb09B8EfC1832c73c09b5Ecf59)
+- Ben's free [startup feedback sessions](https://werd.io/2020/free-startup-feedback-sessions-for-members) (only for members!)
+- Stake for the [HackMoney hackathon](https://hack.ethglobal.co/online/hacker)
+- Unlock Jeremy Epstein marketing blog!
```

Comment:
I think we need a link on this one

Reply:
yeah I am waiting to hear from him. But I guess I'll remove if I don't hear from him this morning.